ci-operator/step-registry/cucushift/e2e/devel/cucushift-e2e-devel-commands.sh

```diff
@@ -4,6 +4,23 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+trap 'warn_0_case_executed' INT TERM EXIT
+function warn_0_case_executed {
+    local count
+    count="$(ls ${ARTIFACT_DIR} | wc -l)"
+    if [ $((count)) == 0 ] ; then
+        mkdir --parents "${ARTIFACT_DIR}"
+        cat >"${ARTIFACT_DIR}/junit-cucushift-result.xml" <<- EOF
+<testsuite name="cucushift-e2e" tests="1" errors="1">
```

Comment:
I don't remember how we used to catch the junits into the proper test suites. Is this going to the Overall CI test suite?

Reply:
This one only exist when there are no test cases executed, which means you will see the general test cases, or only this one.

Reply:
ok, thanks for clarifying @liangxia